build.rs

```diff
@@ -11,6 +11,8 @@ use std::process::Stdio;
 use which::which;
 
 fn main() {
+  println!("cargo:rerun-if-changed=src/binding.cc");
```

Comment:
This is a minor bug fix. Previously, changes to binding.cc wouldn't always get picked up.